src/chi/components/input-text/_input-wrapper.scss

```diff
@@ -15,18 +15,34 @@
             width: 1rem;
           }
 
+          & ~ .a-spinner {
+            position: absolute;
+            top: 0.5rem;
+            right: 0.5rem;
+          }
+
           &.-small {
             & ~ .a-icon {
               height: 0.75rem;
               top: 0.375rem;
               width: 0.75rem;
             }
+
+            & ~ .a-spinner {
+              top: 0.3rem;
+              right: 0.5rem
+            }
           }
 
           &.-large {
             & ~ .a-icon {
               top: 0.75rem;
             }
+
+            & ~ .a-spinner {
```

Comment:
@davidojedacl remove this class as it inherits the properties from default .a-spinner